src/gc/gcimpl.h

```diff
@@ -274,6 +274,13 @@ class GCHeap : public IGCHeapInternal
 
     virtual void DescrGenerationsToProfiler (gen_walk_fn fn, void *context);
 
+    virtual void SetDirty(void* address, size_t size);
+
+    virtual void* GetTable();
```

Comment:
This should be though through with the rest of the methods to gather write barrier geometry: type of write barrier, low/high, ephemeral low/high, card table location, etc. Having one method for each of these does not sound right. Instead, I think we should have a callback on the GCToEEInterface that takes a struct with all write barrier parameters, that the GC will call whenever the write barrier needs updated. This will also replace the current StompWriteBarrierEphemeral, StompWriteBarrierResize that do not go through the abstraction yet.